test/src/test/java/org/corfudb/integration/LogReplicationReaderWriterIT.java

```diff
@@ -277,7 +279,9 @@ public static void writeSnapshotMsgs(List<LogReplicationEntryMsg> msgQ, CorfuRun
         ReplicationSession replicationSession =
             ReplicationSession.getDefaultReplicationSessionForCluster(SINK_CLUSTER_ID);
 
-        StreamsSnapshotWriter writer = new StreamsSnapshotWriter(rt, configManager, logReplicationMetadataManager,
+        LogReplicationContext replicationContext = new LogReplicationContext(configManager, 0, DEFAULT_ENDPOINT);
```

Comment:
you can create a global replicationContext variable instead of creating one every time

Reply:
I think for this IT we are using different runtimes to create configManager and further pass it to context, so we cannot extract it to be a global variable